scripts/lib/CIME/XML/generic_xml.py

```diff
@@ -461,3 +461,11 @@ def get_id(self):
         if xmlid is not None:
             return xmlid
         return self.name(self.root)
+
+    def elements_equal(self, other):
+        if e1.tag != e2.tag: return False
```

Comment:
There's probably an easier way to do this. Do XML Elements have a built-in equality operator?

Reply:
Looks like _Element in this file already supports equality operations.

Reply:
so I only need to remove the new one that I introduced?

Reply:
Yes, just use elem1 == elem2